index.js

```diff
@@ -29,6 +44,8 @@ const removeProperty = (to, from, property) => {
 const shouldCopyProperty = property => property !== 'length';
 
 const mimicFn = (to, from) => {
+	const {name} = to;
```

Comment:
`to.name` is being modified by the next lines, so we need to keep a reference to the initial value here.